### DISCUSSION TITLE
Issues with !pip install pyyaml==5.1 (in google colab)

### DISCUSSION THREAD

**Discussion by Frn1nd0** [facebookresearch/detectron2#5123] [General]
Hi everyone, I got issues with running this code in google colab:

!python -m pip install pyyaml==5.1
import sys, os, distutils.core
!git clone 'https://github.com/facebookresearch/detectron2'
dist = distutils.core.run_setup("./detectron2/setup.py")
!python -m pip install {' '.join([f"'{x}'" for x in dist.install_requires])}
sys.path.insert(0, os.path.abspath('./detectron2'))

The output always show:

Collecting pyyaml==5.1
Downloading PyYAML-5.1.tar.gz (274 kB)
━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 274.2/274.2 kB 5.0 MB/s eta 0:00:00
error: subprocess-exited-with-error

× python setup.py egg_info did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
Preparing metadata (setup.py) ... error
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
Cloning into 'detectron2'...

I am still new to python, detectron2, object detection, all these areas, so, I have no idea on what is happening here, can someone please guide me? Thanks

**Comment by enriquegv001** [comment]
Did you find the solution?
